### PR TITLE
lots of miniscule fixes

### PR DESCRIPTION
**Pull request by iCurbix**
these changes include:
 - fixed several typos, grammar errors, inconsistencies in the documentation
 - fixed several incorrect links in the docs
 - added missing code snippets for typescript in the docs
 - fixed some of the existing code snippets containing errors and made some non-critical improvements (both python and ts)
 - removed empty code blocks existing at the bottom of the docs’ pages
 - reflected recommendation about not sending identifying users’ information to 3rd parties in the contents of the docs
discovered a few bugs with web app behaviour
 - updated ts code snippets to match changes to sdk which are currently in review

### CODE REVIEW

File: evaluation/user-feedback.mdx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+{/* TODO: this page is basically the same thing as monitoring/collect-user-feedback. not sure what to do with it */}
```
Comment (maciej-trebacz) [inline evaluation/user-feedback.mdx:1]: Good point, let's remove this page entirely and keep the `monitoring/collect-user-feedback` but only after we merge these two sections into one (rename `Instructions` to `Capturing Annotations` and use the example code from the evaluation page because it shows how to use other methods like `log`.

Note: Make sure to update all links to this page in the docs to reflect this change, and create a task for Melville to update links (if there are any) in the app too.

image

File: monitoring/advanced-tracing.mdx [hunk 2]
```diff
@@ -48,13 +49,11 @@ def ask_question(question="What is the capital of the US?") -> str:
 ````
```
Comment (maciej-trebacz) [inline monitoring/advanced-tracing.mdx:49]: Is there one too many backticks here? Same in the section below.

File: monitoring/tracing.mdx [hunk 3]
```diff
@@ -40,8 +40,9 @@ For example, you are creating a AI travel agent to assist with booking trips. Th
 
 ### Instruction
 
-If you haven't authorize Baserun, please refer to the [getting started with SDK > Authentication](/uthentication) section for setup instructions.
+If you haven't authorize Baserun, please refer to the [getting started with SDK > Authentication](/getstartedwithSDK/authentication) section for setup instructions.
 
+{/* TODO: What does this refer to? The code doesn't show choosing any kind of environment */}
```
Comment (maciej-trebacz) [inline monitoring/tracing.mdx:45]: We should check show does the SDK decide which environment it's running in, and put that into the documentation. I'm not 100% sure but it might look at the `ENV` environment variable for that.

File: monitoring/users-and-sessions.mdx [hunk 4]
```diff
@@ -48,137 +54,139 @@ For the Use cases of a chat bot, a Session may be the entire conversation. For a
...
 
 ### Continuing an existing session
 
-Using the Session ID that was collected when the session was started you can then resume that session by providing the session ID to the `with_session` context manager.
+Using the Session ID that was collected when the session was started you can then resume that session by providing the session ID.
 
 ### Full example
 
 <CodeGroup>
+{/* TODO: python code does more complex things than ts code. should it be simplified to match? */}
```
Comment (maciej-trebacz) [inline monitoring/users-and-sessions.mdx:107]: Yes, let's make sure both code examples do the same thing, the examples should be minimal examples of the features described above, nothing else.

File: monitoring/advanced-tracing.mdx [hunk 5]
```diff
@@ -1,28 +1,29 @@
...
 sidebarTitle: Advanced tracing features
-description: Tracing additional data such as trace name, custom log, session ID, etc
+description: Tracing additional data such as trace name, custom log, session ID, etc.
 ---
 
 This page goes over some advanced tracing features.
 
 ## Adding a trace name
-
+{/* TODO: this section baad. I don't know how to fix it as I don't know what intention is here, what we actually want to show or explain */}
```
Comment (maciej-trebacz) [inline monitoring/advanced-tracing.mdx:10]: I think this is a valid section, because if you don't add the name parameter you'll end up with a trace list that looks like this: 
image

I would change the description above the code examples to match both Python and TS code though, and maybe change the name parameter value to match what's on the screenshot below the code samples.

File: monitoring/advanced-tracing.mdx [hunk 6]
```diff
@@ -86,8 +85,9 @@ def ask_question(question="What is the capital of the US?") -> str:
 
 ```
 
+{/* don't know how to make it do the same thing as python code, where you put some result in the metadata, without bigger changes in how one is supposed to use baserun.trace from ts sdk */}
```
Comment (maciej-trebacz) [inline monitoring/advanced-tracing.mdx:88]: Ok, I think we need to add a ticket to fix the TS SDK to match what Python does and in the meantime let's leave the docs as they are now.

File: monitoring/tracing.mdx (hunk 3 above)
Comment (iCurbix) [reply monitoring/tracing.mdx:45]: From what I can see in js and python sdks it looks like runs are marked to be either "TEST" or "PRODUCTION" but I have no idea what difference that makes. But that's not even something that you as a user have a direct control of. It kind of makes a guess if the code is run inside a test e.g. by pytest.

So...I still have no idea what this is about. Maybe it used to work differently and docs were not updated? No clue.

File: evaluation/user-feedback.mdx [hunk 7]
```diff
@@ -1,60 +0,0 @@
-
```
Comment (erik-megarad) [inline evaluation/user-feedback.mdx:1]: Why was this file deleted?

**Review by erik-megarad — CHANGES_REQUESTED** [review]
Looks good except maybe the user-feedback page

File: evaluation/user-feedback.mdx (hunk 7 above)
Comment (erik-megarad) [reply evaluation/user-feedback.mdx:1]: Yes, we want to keep it so we have the menu item that might be exactly what a user is looking for.

Comment (iCurbix) [reply evaluation/user-feedback.mdx:1]: how about like so? I made a "snippet" out of the content of the page and put it in both monitoring/collect-user-feedback and to restored evaluation/user-feedback. this way we have both pages but they just display the same thing
